contrib/babelfishpg_common/src/datetimeoffset.c

```diff
@@ -516,7 +516,7 @@ datetimeoffset_smalldatetime(PG_FUNCTION_ARGS)
 	tsql_datetimeoffset *df = PG_GETARG_DATETIMEOFFSET(0);
 	Timestamp	result;
 
-	datetimeoffset_timestamp_internal(df, &result);
+	result = df->tsql_ts;
```

Comment:
Thought: Makes me wonder about initially why we added this logic ;/
Can we check if the original tests which have been added with this change are still passing and if not, then verify if the current output is actually the expected one.

Reply:
So all test file output which I have changed in my PR has been verified and is correct .